NewDawn/NewDawnTests/NewDawnTests.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import XCTest
-@testable import NewDawn
+// @testable import NewDawn
```

Comment:
Why change this?

Reply:
It throws me error about minimum deployment target. Will resolve that later.